ssc/cmod_fresnel_physical_iph.cpp

```diff
@@ -1087,7 +1108,7 @@ class cm_fresnel_physical_iph : public compute_module
         // Instantiate Solver
         C_csp_solver csp_solver(weather_reader,
             c_fresnel,
-            c_heat_sink,
+            *c_heat_sink_pointer,
```

Comment:
Is this checked for null now that a pointer is used?

Reply:
Added nullptr check ([850d489](https://github.com/NREL/ssc/pull/1250/commits/850d48936bb946817e914ceddf7b38d4df072f16))